minvws/nl-kat-coordination
# Add some additional exception handling to the scheduler

Changes:
* Added pydantic response model to octopoes. This will parse the paginated response from octopoes which will throw the correct `ValidationError` exception.
* Added `ExternalServicesException` that can be catched by the rest of the application and handle it correctly.

QA:

- Start kat
- Kill octopoes (docker-compose kill octopoes)
- Check scheduler logs
- Scheduler should not be terminated because of a faulty response from octopoes

Closes #2768
Closes #2769
